core/src/display_object/edit_text.rs

```diff
@@ -1681,6 +1681,21 @@ impl<'gc> TDisplayObject<'gc> for EditText<'gc> {
         self.0.write(context.gc_context).object = Some(to.into());
     }
 
+    fn set_parent(&self, context: &mut UpdateContext<'_, 'gc>, parent: Option<DisplayObject<'gc>>) {
+        let had_parent = self.parent().is_some();
+        self.base_mut(context.gc_context)
+            .set_parent_ignoring_orphan_list(parent);
+        let has_parent = self.parent().is_some();
```

Comment:
Nit: Is there any scenario where `parent` != `self.parent()` on this line? Save's us a bit of indirection 😅